### ISSUE TITLE
Make opportunity from communication will also add a lead for existing contacts

### ISSUE THREAD

**Issue by basdereus84** [frappe/erpnext#12665]
When manual making an opportunity from communication (email) by clicking "Make, Opportunity", a lead will be created even if the communication is linked to an existing contact.

When the system automatically appends communication (email) to an opportunity (as configured in the email account settings), it's working correctly. An unknown sender is converted and linked to a (new) lead and then a new opportunity is created and linked to that lead. If the sender is an existing contact, a new opportunity is created and linked to the contact. 

The same goes for creating an issue.

I don't think this related to https://github.com/frappe/erpnext/issues/12666 as I'm linking the communication manual to the correct contact before converting it into an opportunity or issue.

**To sum up:** Automatically appending an email to an opportunity or issue works as expected. Doing the same thing manually, for instance from an email inbox, doesn't
